justfile

```diff
@@ -1,18 +1,43 @@
+# Function to check and remove the .movement/ directory if it exists
+remove-dot-movement:
+    @if [ -d .movement ]; then \
+        echo "Removing .movement directory..."; \
+        rm -rf .movement; \
+    fi
+
```

Comment:
Would it make sense to also remove `.data` if it's present, or is that not so consequential?